src/QuadForm/Types.jl

```diff
@@ -1,3 +1,35 @@
+export AbstractLat
+export AbstractSpace
+export AbstractSpaceMor
+export AbstractSpaceRes
+export HermGenus
+export HermLat
+export HermLocalGenus
+export HermSpace
+export JorDec
+export LatCloseEnumCtx
+export LineEnumCtx
```

Comment:
```suggestion
```
We do not want to make these types part of the interface. They are internal.
Were they exported before?

Reply:
They were not, but if I do not export them, will the algorithms in the other files detect the type definitions ?

Reply:
Everything resides in the `module Hecke`, and there every type/function is visible to everyone. One just has to be careful about the order of inclusion.